doc/classes/ProjectSettings.xml

```diff
@@ -304,6 +304,12 @@
 		<member name="audio/driver/output_latency.web" type="int" setter="" getter="" default="50">
 			Safer override for [member audio/driver/output_latency] in the Web platform, to avoid audio issues especially on mobile devices.
 		</member>
+		<member name="audio/general/2d_panning_strength" type="float" setter="" getter="" default="0.5">
+			The strength of the panning effect for AudioStreamPlayer2D nodes.
```

Comment:
This project setting doesn't seem to be read when it's changed at run-time, so the value can't be changed at run-time. This should be made possible to make forcing non-positional audio output easier for accessibility. This is not critical for a first iteration of this feature, but in this case, the limitation should be documented in the property description.

See https://github.com/godotengine/godot/pull/53296#pullrequestreview-877227747 which added this signal in `3.x` for reference (it's also available in `master`).

Reply:
I set it up as a property that requires restart, because I couldn't figure out how to do runtime cache invalidation for project setting caches. The accessibility use-case can be addressed with the use of the Stereo Enhance effect as discussed in #58823

I'll address the rest of these in a bit!

Reply:
You can use `project_settings_changed` signal for cache invalidation.